private/buf/buffetch/internal/internal.go

```diff
@@ -499,35 +499,41 @@ func NewWriter(
 // The Path will be the only value set when the RawRefProcessor is invoked, and is not normalized.
 // After the RawRefProcessor is called, options will be parsed.
 type RawRef struct {
-	// Will always be set
-	// Not normalized yet
+	// Will always be set.
+	// Not normalized yet.
 	Path string
-	// Will always be set
-	// Set via RawRefProcessor if not explicitly set
+	// Will always be set.
+	// Set via RawRefProcessor if not explicitly set.
 	Format string
-	// Only set for single, archive formats
-	// Cannot be set for zip archives
+	// Only set for single, archive formats.
+	// Cannot be set for zip archives.
 	CompressionType CompressionType
-	// Only set for archive, git formats
+	// Only set for archive, git formats.
 	SubDirPath string
-	// Only set for git formats
-	// Only one of GitBranch and GitTag will be set
+	// Only set for git formats.
+	// Only one of GitBranch and GitCommit will be set.
 	GitBranch string
-	// Only set for git formats
-	// Only one of GitBranch and GitTag will be set
-	GitTag string
-	// Only set for git formats
-	// Specifies an exact git reference to use with git checkout.
-	// Can be used on its own or with GitBranch. Not allowed with GitTag.
-	// This is defined as anything that can be given to git checkout.
+	// Only set for git formats.
+	// Only one of GitBranch and GitCommit will be set.
+	// Should indicate a full commit hash or tag name.
+	// This is defined as anything that can be given to "git fetch".
+	GitCommit string
```

Comment:
This rename seems to just go in the other direction - now, in the code I would be confused as to why a tag can be used here. I would suggest naming this `GitCommitOrTag`, and updating the functions in this PR appropriately per that naming

Reply:
Renamed